test/load_tests/2-cdos.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+./main.py -u http://192.168.24.43/api/ reset dos_init > 0.log
```

Comment:
Would it make sense to replace these scripts with scripts that take a command line parameter for the hostname?

Reply:
A parameter would be better. Or my approach in autotest2.sh is to pass an environment variable, $URL.